project/sherLOCKED/Assets/Scripts/Menus/MainMenuController.cs

```diff
@@ -12,6 +12,8 @@ public class MainMenuController : MonoBehaviour
     public GameObject customizeMenu;
     public GameObject creditsMenu;
 
+    public GameObject[] buttons;
+
```

Comment:
Not sure why this commit is making changes in these files?

Reply:
neither do i - it happened during the reset ?

Reply:
Alright, see if you can reset them to what's on origin/master, we should try and avoid mixing commits too much